src/renderers/dom/shared/devtools/ReactDOMUnknownPropertyDevtool.js

```diff
@@ -22,6 +22,15 @@ if (__DEV__) {
     dangerouslySetInnerHTML: true,
     key: true,
     ref: true,
+
+    defaultValue: true,
+    valueLink: true,
+    defaultChecked: true,
+    checkedLink: true,
+    innerHTML: true,
+    suppressContentEditableWarning: true,
+    onFocusIn: true,
+    onFocusOut: true,
```

Comment:
suppressContentEditableWarning seems okay, but what is innerHTML here for? and onFocusIn and onFocusOut? I also don't love that the form props are here but I guess it's not terrible.